internal/services/eventhub/eventhub_namespace_data_source.go

```diff
@@ -111,6 +107,15 @@ func EventHubNamespaceDataSource() *pluginsdk.Resource {
 			"tags": commonschema.TagsDataSource(),
 		},
 	}
+
+	if !features.FourPointOhBeta() {
+		resource.Schema["zone_redundant"] = &pluginsdk.Schema{
+			Type:     pluginsdk.TypeBool,
+			Computed: true,
+		}
```

Comment:
we should still add a deprecation message here?

Reply:
Fixed.